Tools/CISupport/ews-build/steps.py

```diff
@@ -294,9 +305,25 @@ def add_patch_id_url(self):
             self.addURL('Patch {}'.format(patch_id), Bugzilla.patch_url(patch_id))
 
     def add_pr_details(self):
+        repository_url = self.getProperty('repository', '')
         pr_number = self.getProperty('github.number')
-        if pr_number:
-            self.addURL('Pull request {}'.format(pr_number), GitHub.pr_url(pr_number=pr_number, repository_url=self.getProperty('repository')) or '')
+        title = self.getProperty('github.title', '')
+        owners = self.getProperty('owners', [])
+        revision = self.getProperty('revision')
+
+        if pr_number and title:
+            self.addURL('[PR {}] {}'.format(pr_number, title or '?'), GitHub.pr_url(pr_number, repository_url))
+        if pr_number and revision:
+            self.addURL('{}'.format(revision), GitHub.commit_url(revision, repository_url))
+        if owners:
+            contributors, errors = Contributors.load(use_network=False)
+            for error in errors:
+                print(error)
+                self._addToLog('stdio', error)
```

Comment:
This class (ConfigureBuild) doesn't seem to have _addToLog() method. Might need to add that.